elife/scripts/rds-perms2.sh

```diff
@@ -0,0 +1,42 @@
+#!/bin/bash
```

Comment:
Related: it seems this script is run also for local Postgresql databases, not for RDS only. So maybe change the `rds-` prefix?